modules/core/include/visp3/core/vpImageConvert.h

```diff
@@ -64,12 +64,17 @@
 #include <windows.h>
 #endif
 
-#if defined(VISP_HAVE_PCL) && defined(VISP_HAVE_THREADS)
+#if defined(VISP_HAVE_PCL) && defined(VISP_HAVE_PCL_COMMON) && defined(VISP_HAVE_THREADS)
 #include <visp3/core/vpColVector.h>
 #include <visp3/core/vpImageException.h>
 #include <visp3/core/vpPixelMeterConversion.h>
 
+#include <pcl/pcl_config.h>
+#if PCL_VERSION_COMPARE(>=,1,14,1)
 #include <pcl/impl/point_types.hpp>
+#else
+#include <pcl/point_types.h>
+#endif
```

Comment:
Same here. Only `#include <pcl/point_types.h>` ?

Reply:
I hadn't noticed that. Modified in 3329f0310